engine/app/components/citizens_advice_components/header.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+module CitizensAdviceComponents
+  class Header < Base
+    renders_many :skip_links, "SkipLink"
+    renders_many :header_links, "HeaderLink"
+
+    renders_one :account_link, "AccountLink"
+
+    renders_one :logo, lambda { |title:, url: "/"|
```

Comment:
Might get caught up on rubocop / linting but usually stabby lambdas use the `{}` and long ones use `do end`

i.e. 
```
renders_one :logo, -> (title, url) { puts title + url }
```

```
renders_one :logo, lambda do |title, url|
  puts title + url
end
```

Reply:
Yeah I had this the second way but rubocop complained!

Reply:
Interesting. It shouldn't have done, but I may have missed some config we might have setup in the style repo.... I'll have a quick look at this.

Reply:
In fact trying the second way throws a `tried to create Proc object without a block` error.

Reply:
I "think" this might be a library / ruby 2.7 interop bug. But I'm not 100% sure. This "should" work / be interchangeable, I'll look into this for my own learnings but feel free to disregard for now if you want.